### ISSUE TITLE
Logically nest header elements

### ISSUE THREAD

**Issue by brainwane** [OpenTechStrategies/psm#512]
In a few places our header nesting is wrong, causing accessibility issues (which `pa11y` calls out as `Section508.D.HeadingOrder`). Examples where an `h3` should be an `h2`:

* `psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/screening_errors.jsp`:        `<h3>The following problems were identified for this enrollment during screening:</h3>`
* the Print Preview `h3` in the Provider Dashboard
